platform/ui/src/components/Tooltip/PortalTooltip.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+import Card from './PortalTooltipCard';
+
+const portalNodes = {};
+
+/**
+ * A portal based tooltip.
+ */
+export default class ToolTip extends React.Component {
```

Comment:
I sort of expected this to be named `PortalTooltip` to match the name of the file. I think I prefer it being named `PortalTooltip` but maybe I am being picky. Also note that the other/existing tool tip component is named `Tooltip` (i.e. small t for tip). We should be consistent one way or the other.